app/app.go

```diff
@@ -230,16 +230,18 @@ var (
 
 	// module account permissions
 	maccPerms = map[string][]string{
-		authtypes.FeeCollectorName:               nil,
-		distrtypes.ModuleName:                    nil,
-		stakingtypes.BondedPoolName:              {authtypes.Burner, authtypes.Staking},
-		stakingtypes.NotBondedPoolName:           {authtypes.Burner, authtypes.Staking},
-		govtypes.ModuleName:                      {authtypes.Burner},
-		ibctransfertypes.ModuleName:              {authtypes.Burner},
-		subscriptionmoduletypes.ModuleName:       {authtypes.Burner, authtypes.Staking},
-		dualstakingmoduletypes.BondedPoolName:    {authtypes.Burner, authtypes.Staking},
-		dualstakingmoduletypes.NotBondedPoolName: {authtypes.Burner, authtypes.Staking},
-		pairingmoduletypes.ModuleName:            {authtypes.Burner, authtypes.Staking},
+		authtypes.FeeCollectorName:                                       nil,
+		distrtypes.ModuleName:                                            nil,
+		stakingtypes.BondedPoolName:                                      {authtypes.Burner, authtypes.Staking},
+		stakingtypes.NotBondedPoolName:                                   {authtypes.Burner, authtypes.Staking},
+		govtypes.ModuleName:                                              {authtypes.Burner},
+		ibctransfertypes.ModuleName:                                      {authtypes.Burner},
+		subscriptionmoduletypes.ModuleName:                               {authtypes.Burner, authtypes.Staking},
+		dualstakingmoduletypes.BondedPoolName:                            {authtypes.Burner, authtypes.Staking},
```

Comment:
remove dualstaking and pairing pools

Reply:
done. see https://github.com/lavanet/lava/commit/3c190e65adf95d112a07287483dce09123539c1a